@here/harp-mapview/lib/MapView.ts

```diff
@@ -548,6 +548,18 @@ export interface MapViewOptions extends TextElementsRendererOptions, Partial<Loo
      */
     languages?: string[];
 
+    /**
+     * Sets the data sources to use specific country point of view (political view).
+     *
+     * This option may result in rendering different country borders then commonly accepted for
+     * some regions and it mainly regards to so called __disputed borders__. Although not all
+     * data sources or themes may support it.
+     *
+     * @note Country code should be coded in ISO 3166-1 alpha-2 standard, if this option
```

Comment:
`ISO 3166-1 alpha-2` (https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) uses upper case codes everywhere.

This document: https://github.com/tilezen/vector-datasource/blob/master/docs/layers.md says "lower-case ISO 3166-1 alpha-2 codes or the pseudo-code iso" ...

Reply:
Ok, good catch, I will update the documentation that it is lowercase.